sdk/ai/azure-ai-inference/azure/ai/inference/_patch.py

```diff
@@ -102,8 +102,8 @@ def load_client(
             "The AI model information is missing a value for `model type`. Cannot create an appropriate client."
```

Comment:
@M-Hietala could you please also take care of https://github.com/Azure/azure-sdk-for-python/issues/37729 while you're on it?

Reply:
if you are referring to the distributed_trace then i already tried removing it, but the ci will fail in that case complaining something about all functions making http requests need to have it. if you know that that does not apply here and know how to disable to error, please let me know.